distutils/command/install.py

```diff
@@ -411,6 +411,10 @@ def finalize_options(self):
                             'implementation_lower': _get_implementation().lower(),
                             'implementation': _get_implementation(),
                            }
+        try:
+            local_vars['py_version_nodot_plat'] = sys.winver.replace('.', '')
+        except AttributeError:
+            local_vars['py_version_nodot_plat'] = ''
```

Comment:
Same here - is this value reliably set in sysconfig.get_config_vars()? If so, it should be possible to omit this work.

Reply:
This value was only added in Python 3.10, so is needed at least for older Pythons.